Epic Games not launching in sandboxie, terminating its services instantly

### Describe what you noticed and did

soo im facing problem with lauching EGL=epic games launcher, and not running its services (EpicWebHelper)
i tried so many times and so many fixes and yet nothing works, i asked the AI and he told me its either windows security conflict or EGLnew update issue


### How often did you encounter it so far?

several times, 1 time was 2 days ago

### Expected behavior

instant termination, no EpicWebHelper service (used to run before), reloading sandbox config (by itself), sandbox shut down immediatly

### Affected program

latest version of Epic Games, and its games 

### Download link

https://store.epicgames.com/en-US

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Did the program or any related process close unexpectedly?

Yes, it did, but I don't know how to attach the .dmp file(s).

### Crash dump

_No response_

### What version of Sandboxie are you running now?

sanboxie V1.17.2

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### Is it a regression from previous versions?

all version wont start the app

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

Win11 Pro H25

### In which Windows account you have this problem?

A local account (Standard user).

### Please mention any installed security software

MDA only

### Did you previously enable some security policy settings outside Sandboxie?

no i didnt
the problem started Unexpectedly, in the morning of the first day i was able to play, but in the night the problem started

### Trace log

_No response_

### Sandboxie.ini configuration

```ini
[New_Box]
Enabled=y
BlockNetworkFiles=y
RecoverFolder=%Desktop%
RecoverFolder=%Personal%
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
BorderColor=#02f6f6,ttl
Template=AutoRecoverIgnore
Template=LingerPrograms
Template=BlockPorts
Template=qWave
Template=FileCopy
Template=SkipHook
Template=OpenBluetooth
ConfigLevel=10
UseFileDeleteV2=y
UseRegDeleteV2=y
AutoRecover=y
```

I found that it is this EOSBootStrapper.exe that blocks Epic Games launching.
It creates a log like this:
Noted the [ERROR] in last three lines

```
[2026-04-11 11:23:10.513] [Info] [FGenericBootstrapper::Bootstrap] Started Epic Online Services Bootstrapper ++EOSSDK+Main-0.0.0.0.
[2026-04-11 11:23:10.525] [Verbose] [FGenericBootstrapper::GetConfigPath] Determined config path: 'C:\Program Files\Epic Games\Launcher\Portal\Extras\EOSBootStrapper\EOSBootStrapper.ini'.
[2026-04-11 11:23:10.527] [Info] [FGenericBootstrapper::Bootstrap] Reading config file: 'C:\Program Files\Epic Games\Launcher\Portal\Extras\EOSBootStrapper\EOSBootStrapper.ini'.
[2026-04-11 11:23:10.529] [Info] [FGenericBootstrapper::TryReadConfig] Reading config file: 'C:\Program Files\Epic Games\Launcher\Portal\Extras\EOSBootStrapper\EOSBootStrapper.ini'.
[2026-04-11 11:23:10.530] [Verbose] [FGenericPlatform::TryReadFile] Trying to read file at path: 'C:\Program Files\Epic Games\Launcher\Portal\Extras\EOSBootStrapper\EOSBootStrapper.ini'
[2026-04-11 11:23:10.535] [Verbose] [FGenericPlatform::TryReadFile] Successfully read file: 'C:\Program Files\Epic Games\Launcher\Portal\Extras\EOSBootStrapper\EOSBootStrapper.ini'
[2026-04-11 11:23:10.537] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing line: 'ApplicationPath=..\..\Binaries\Win64\EpicGamesLauncher.exe'.
[2026-04-11 11:23:10.539] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing key: 'ApplicationPath'.
[2026-04-11 11:23:10.542] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing Value: '..\..\Binaries\Win64\EpicGamesLauncher.exe'.
[2026-04-11 11:23:10.544] [Info] [FGenericBootstrapper::TryReadConfig::<lambda_1>::operator ()] Set: 'ApplicationPath=C:\Program Files\Epic Games\Launcher\Portal\Binaries\Win64\EpicGamesLauncher.exe'.
[2026-04-11 11:23:10.547] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing line: 'WorkingDirectory='.
[2026-04-11 11:23:10.549] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing key: 'WorkingDirectory'.
[2026-04-11 11:23:10.551] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing line: 'ClientDirectory=*'.
[2026-04-11 11:23:10.552] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing key: 'ClientDirectory'.
[2026-04-11 11:23:10.554] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing Value: '*'.
[2026-04-11 11:23:10.556] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing line: 'WaitForExit=0'.
[2026-04-11 11:23:10.557] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing key: 'WaitForExit'.
[2026-04-11 11:23:10.559] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing Value: '0'.
[2026-04-11 11:23:10.561] [Info] [FGenericBootstrapper::TryReadConfig::<lambda_2>::operator ()] Set: 'WaitForExit=false'.
[2026-04-11 11:23:10.564] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing line: 'SkipEOS=0'.
[2026-04-11 11:23:10.566] [Verbose] [FGenericBootstrapper::TryReadConfig] Parsing key: 'SkipEOS'.
[2026-04-11 11:23:10.569] [Error] [FGenericBootstrapper::TryReadConfig] Unrecognized key ignored!
[2026-04-11 11:23:10.571] [Info] [FGenericBootstrapper::TryReadConfig] Successfully read config.
[2026-04-11 11:23:10.577] [Verbose] [FWindowsPlatform::TryCreateGuid] Successfully created guid components: 0x57B1B75B 0x5DA2407A 0x9C30096D 0x8E550023
[2026-04-11 11:23:10.579] [Verbose] [FGenericBootstrapper::TryCreateSession] Generated Session GUID: '57b1b75b-5da2-407a-9c30-096d8e550023'.
[2026-04-11 11:23:10.582] [Verbose] [FWindowsPlatform::TryCreateProcessGroup] Trying to create job object with name: 'Global\57b1b75b-5da2-407a-9c30-096d8e550023'
[2026-04-11 11:23:10.588] [Error] [FWindowsPlatform::LogErrorWithLastErrorMessage] Failed to query job object limit information: ErrorCode=0x5 (5), ErrorMessage=存取被拒。 
[2026-04-11 11:23:10.591] [Error] [FGenericBootstrapper::TryCreateSession] Failed to create a new process group.
[2026-04-11 11:23:10.593] [Error] [FGenericBootstrapper::Bootstrap] Failed to create a new session!
```

--- a/Sandboxie/core/dll/sysinfo.c
+++ b/Sandboxie/core/dll/sysinfo.c
@@ -556,6 +556,17 @@ _FX NTSTATUS SysInfo_GetJobName(OBJECT_ATTRIBUTES* ObjectAttributes, WCHAR** Out
     if (ObjectAttributes && ObjectAttributes->ObjectName) {
         objname_len = ObjectAttributes->ObjectName->Length / sizeof(WCHAR);
         objname_buf = ObjectAttributes->ObjectName->Buffer;
+
+        // Normalize Win32 BaseNamedObjects prefixes to plain names before
+        // constructing the sandbox object path.
+        if (objname_len >= 7 && _wcsnicmp(objname_buf, L"Global\\", 7) == 0) {
+            objname_len -= 7;
+            objname_buf += 7;
+        }
+        else if (objname_len >= 6 && _wcsnicmp(objname_buf, L"Local\\", 6) == 0) {
+            objname_len -= 6;
+            objname_buf += 6;
+        }
     } 
     else { // unnamed job
 

